agent.go

```diff
@@ -480,6 +504,80 @@ func (p *pod) closeChannels() {
 	}
 }
 
+// listenToUdevEvents listens to udev events to detect device nodes added
+// and binds the device nodes to the /dev of all the containers.
+// This is required as a container has its /dev mounted with tmpfs with an initial
+// list of devices.
```

Comment:
It would be good to call out when this happens. I assume this is for the use case where devices are specified as part of the container config.json and you are hot plugging them into the VM, and now expect them to show up in the specific container?

Reply:
Or is this due to a device creation due to some action in the container which has the CAPs to perform device creation, and this device is created due to an activity in the container itself does not show up in the container.

Reply:
@mcastelino It is the latter case. When you are performing some action in the container that results in a device node being created in the VM. This is useful for running DPDK inside the container where you would typically bind a device to vfio or uio resulting in a new device node being created.